curiosrc/seal/task_movestorage.go

```diff
@@ -136,18 +136,39 @@ func (m *MoveStorageTask) CanAccept(ids []harmonytask.TaskID, engine *harmonytas
 }
 
 func (m *MoveStorageTask) TypeDetails() harmonytask.TaskTypeDetails {
+	ssize := abi.SectorSize(32 << 30) // todo task details needs taskID to get correct sector size
+	if isDevnet {
+		ssize = abi.SectorSize(2 << 20)
+	}
+
 	return harmonytask.TaskTypeDetails{
 		Max:  m.max,
 		Name: "MoveStorage",
 		Cost: resources.Resources{
-			Cpu: 1,
-			Gpu: 0,
-			Ram: 128 << 20,
+			Cpu:     1,
+			Gpu:     0,
+			Ram:     128 << 20,
+			Storage: m.sc.Storage(m.taskToSector, storiface.FTNone, storiface.FTCache|storiface.FTSealed|storiface.FTUnsealed, ssize, storiface.PathStorage),
```

Comment:
Can this change to read ssize in dynamically so we request the correct size?

Reply:
This is plumbed to HasCapacity so not really sadly.